eng/clipack/Aspire.Cli.linux-arm64.csproj

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <PropertyGroup>
+    <CliRuntime>linux-arm64</CliRuntime>
```

Comment:
No need to block your PR for this, but we may need to add a linux-musl build as well. Thoughts @mitchdenny @DamianEdwards @davidfowl

Reply:
I think that @danegsta was adding in/has added in the musl support on the DCP side too.

Reply:
This is a new project though, so he wouldn't have done it for this one.

Reply:
Well there is no point shipping musl if the whole stack doesn't support it :)